### ISSUE TITLE
[Feature Request]: SD3 support

### ISSUE THREAD

**Issue by thekingbalXD** [lllyasviel/Fooocus#3140]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do?

Generating image with the new SD3 model

### Proposed workflow

Work as usual

**Comment by Haapavuo** [comment]
Definitely a must have!

**Comment by mirek190** [comment]
bump!

**Comment by UXVirtual** [comment]
From some initial tests it seems like SD3 can follow detailed instructions in prompts a lot closer than in SDXL, especially around adding specific colors to items in the scene (e.g. pieces of clothing). This could work well with Fooocus styles, given that there aren't many Loras available for SD3 yet.

**Comment by mashb1t** [comment]
Please find more information about SD3 support in:

- https://github.com/lllyasviel/Fooocus/discussions/2773#discussioncomment-9643676
- https://github.com/mashb1t/Fooocus/discussions/31#discussioncomment-9474519
- https://github.com/lllyasviel/Fooocus/pull/3084

@UXVirtual SD3 already hinted to some difficulties: SD3 uses a completely different pipeline (mainly encoders, sampling preparation and conditioning), so it's comparable to adding SD 1.5 support with some additional steps, which isn't done quickly and needs patching of ldm_patched first (started in mentioned MR).

As mentioned in https://github.com/lllyasviel/Fooocus/discussions/2336#discussioncomment-9754697 i'm currently working on another feature.

Feel free to support in solving the issues with the [PR](https://github.com/lllyasviel/Fooocus/pull/3084), your help is welcome. @lllyasviel same for you :)

**Comment by ptits** [comment]
SD3 works in Ruined Fooocus

Check some results
https://github.com/runew0lf/RuinedFooocus/issues/145#issue-2354712594

**Comment by IPv6** [comment]
> runew0lf

they have an option to directly use comfyUI behind the scenes without throughout integration with other features. Cool, but... it`s not a Fooocus way

**Comment by mashb1t** [comment]
@IPv6 yes correct, there are various overrides/optimizations/extensions for ComfyUI classes and functions Fooocus uses.

**Comment by blckbx** [comment]
> SD3 works in Ruined Fooocus
> 
> Check some results [runew0lf#145 (comment)](https://github.com/runew0lf/RuinedFooocus/issues/145#issue-2354712594)

Can confirm the observations. Not impressed with SD3. Been playing around with SD3 on StableSwarm for a while. Image quality is good and rich in details. Human anatomy is awful, a big step back imho. I'll stick to SDXL unless there are trained models that fix the shortcomings. Not speaking of the licensing issues and censorship you can read about on civitai etc.
